Source/ImPlotWidgets/Private/HistogramWidget.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+
+#include "ImPlotWidgets/Public/HistogramWidget.h"
+
+#include <ImGui/Public/ImPlotImplementation.h>
+#include <Runtime/Core/Public/Algo/Sort.h>
+
+void SHistogramWidget::Construct(const FArguments& InArgs)
+{
+	SImPlotWidget::Construct(InArgs);
```

Comment:
We can change this to Super now
```suggestion
	Super::Construct(InArgs);
```

Reply:
Another of these in this file, and a couple in the FilledLineGraph widget as well